src/routes/lobby/LobbyView.vue

```diff
@@ -60,6 +60,7 @@
                   color="primary"
                   hide-details
                   @update:model-value="setIsRanked"
+                  @keydown.enter.stop="(e) => e.target.click()"
```

Comment:
I like the simplicity of this solution and that it utilizes the existing click handling. We could use a test case to confirm that it works. Given that another contributor PR'd #1168 to resolve the same issue and they have a test case written, would you be comfortable opening a PR into their fork in the branch [enterToggle/1166](https://github.com/abebanos/cuttle/tree/enterTogggleRanked/1166) to update their implementation of the solution itself to match yours?

I think the winning combination is their test case + your one-line solution. And if possible, I'd like to credit you both for the contribution